Oleksandr_Kotov/8/exceptions.py

```diff
@@ -0,0 +1,35 @@
+
+class TerrainGenerationError(Exception):
+    def __init__(self, scale):
+        self.__scale = scale
+
+    def __str__(self):
+
+        string = "Terrain generaion exception raised."
+        string.join(F" Trying to set scale to {self.__scale}.")
```

Comment:
Have you ever tried this?

Reply:
Changed this method. Interesting behavior. Why does't it concatenates this strings?